docs/release-notes.asciidoc

```diff
@@ -56,6 +57,7 @@ This section summarizes the changes in each release.
 :issue: https://github.com/elastic/kibana/issues/
```

Comment:
I think we can remove this attribute definition, right @joepeeples?

```suggestion
```

Reply:
https://github.com/elastic/security-docs/pull/4976 will take care of removing this definition, so you don't need to do it here, too. Also it might break things if you remove the definition but don't also replace all instances of the old attribute to the global shared attribute, which is the other thing that #4976 does.

Reply:
Oh yeah, gotcha. I'll resolve this and the following comment then.